test/system/test_tpm2_pcrevent.sh

```diff
@@ -58,10 +58,18 @@ tpm2_pcrevent -Q < $hash_in_file
 
 # Test that fifo stdin works
 cat $hash_in_file | tpm2_pcrevent > $hash_out_file
+cat $hash_out_file
```

Comment:
Don't add any superfluous output to the testing code, this cat doesn't add any value, delete it.